src/accel/nesterov.jl

```diff
@@ -1,41 +1,22 @@
-using LinearAlgebra
+struct SimpleNesterovSequence{R} end
```

Comment:
I wonder if the terminology can be kept consistent and use 'NesterovOperator'?

Reply:
Or perhaps use `Acceleration` everywhere?

Reply:
The thing is that there are substantially different "kinds" of objects involved here: some are actual linear operators that you can multiply by, some are sequences that you need to consume somehow. So naming all of them "operator" seems weird to me... Also "acceleration" seems out of context here, since if you look at them individually, these are really linear operators or sequences of numbers, and any acceleration is just a result of how you use them...